new-site/src/components/Hero.scss

```diff
@@ -0,0 +1,149 @@
+@import "./utils.scss";
+
+.hero-container {
+  --koros-height: 145px;
+  --hero-height: 540px;
+  --hero-small-left: calc((100vw - 100%) / 2);
+  --hero-large-content-left: calc((100vw + var(--spacing-s) - var(--container-width-xl)) / 2);
+
+  background-color: var(--color-coat-of-arms);
```

Comment:
We could maybe move this under the media query x-large-and-above-screen, as it seemed to only effect there.

Reply:
- [ ] Move background-color inside media-query